src/main/java/persistence/Application.java

```diff
@@ -21,26 +19,28 @@ public static void main(String[] args) {
         logger.info("Starting application...");
         try {
             final DatabaseServer server = new H2();
+            server.start();
+
             final JdbcTemplate jdbcTemplate = new JdbcTemplate(server.getConnection());
-            final MetamodelInitializer metamodelInitializer = new MetamodelInitializer(jdbcTemplate);
+            final EntityManagerFactory emf = new EntityManagerFactoryImpl(new ThreadLocalCurrentSessionContext(), server);
 
-            server.start();
 
             jdbcTemplate.execute("CREATE TABLE department (department_id BIGINT AUTO_INCREMENT, name VARCHAR(255), PRIMARY KEY (department_id));");
             jdbcTemplate.execute("CREATE TABLE employee (id BIGINT AUTO_INCREMENT, name VARCHAR(255), department_id BIGINT, PRIMARY KEY (id), FOREIGN KEY (department_id) REFERENCES department(department_id));");
```

Comment:
```suggestion
            jdbcTemplate.execute("CREATE TABLE department (department_id BIGINT AUTO_INCREMENT, name VARCHAR(255), PRIMARY KEY (department_id));");
            jdbcTemplate.execute("CREATE TABLE employee (id BIGINT AUTO_INCREMENT, name VARCHAR(255), department_id BIGINT, PRIMARY KEY (id), FOREIGN KEY (department_id) REFERENCES department(department_id));");
```
이런 애들은 어떻게 처리를 하면 좋을까요?

Reply:
```java
                        jdbcTemplate.execute(
                                new CreateTableQueryBuilder(new H2Dialect(), Department.class, em.getMetamodel())
                                        .build()
                        );

                        jdbcTemplate.execute(
                                new CreateTableQueryBuilder(new H2Dialect(), Employee.class, em.getMetamodel())
                                        .build()
                        );
```

만들어둔 QueryBuilder를 활용해보도록 수정하였습니다 :)